src/components/Alert.ts

```diff
@@ -0,0 +1,8 @@
+import { DOM, StatelessComponent } from "react";
+
+export const Alert: StatelessComponent<{ message?: string }> = (props) =>
+    props.message
+        ? DOM.div({ className: "alert alert-danger widget-offline-search-alert" }, props.message)
+        : null as any;
```

Comment:
Get additional prop `className` and pass `widget-offline-search-alert` to keep this component generic